src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixExtension.groovy

```diff
@@ -42,6 +43,11 @@ class ScalafixExtension {
      */
     Boolean autoConfigureSemanticdb = true
 
+    /**
+     * SemanticDB version used by scalafix to run semantic rules.
+     */
+    final Property<String> semanticdbVersion
```

Comment:
sug.: Used to override the version of the SemanticDB compiler plugin. By default this plugin uses a version that is guaranteed to be compatible with Scalafix.